dimod/constrained.py

```diff
@@ -442,6 +444,80 @@ def add_variable(self, v: Variable, vartype: VartypeLike):
         else:
             return self.variables._append(vartype, v)
 
+    @classmethod
+    def from_bqm(cls, bqm: BinaryQuadraticModel) -> 'ConstrainedQuadraticModel':
```

Comment:
Do we want to add `from_binary_quadratic_model` for completeness? :shrug:

Reply:
We'll see if anyone requests it.